ts/Series/OHLC/OHLCSeries.ts

```diff
@@ -126,6 +126,7 @@ class OHLCSeries extends HLCSeries {
 
     public static compose(
         SeriesClass: typeof Series,
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
You should try `varsIgnorePattern` with `^_` pattern instead. https://eslint.org/docs/latest/rules/no-unused-vars#varsignorepattern, so that parameters with underscore get ignored.